src/functions.js

```diff
@@ -626,19 +626,117 @@ defineFunction([
 defineFunction([
     "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
     "\\check", "\\hat", "\\vec", "\\dot",
-    // We don't support expanding accents yet
-    // "\\widetilde", "\\widehat"
+    "\\widehat", "\\widetilde", "\\overrightarrow", "\\overleftarrow",
+    "\\Overrightarrow", "\\overleftrightarrow", "\\overgroup",
+    "\\overlinesegment", "\\overleftharpoon", "\\overrightharpoon",
 ], {
     numArgs: 1,
 }, function(context, args) {
     const base = args[0];
+
+    const isStretchy = !utils.contains([
+        "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+        "\\check", "\\hat", "\\vec", "\\dot",
+    ], context.funcName);
+
+    const isShifty = !isStretchy || utils.contains([
+        "\\widehat", "\\widetilde", "\\overrightarrow", "\\overrightharpoon",
+    ], context.funcName);
```

Comment:
Yes, they are all stretchy. But not all of them are shifty.

The TeXbook, p 443, rule 12, describes that when an accent is placed over a single character, the accent is shifted to the right by the `\skewchar` of its font. And the previously existing accents (i.e. `!isStretchy`) all do that shift.  TeX also shifts `\widehat` and `\widetilde`.

LaTex does not shift `\overrightarrow` or `\overrightharpoon`. I did that on my own because I think they look dopey unshifted. If you want more TeX purity, let me know and I take the shift off those two.